tests/fail/tree_borrows/wildcard/cross_tree_from_main.rs

```diff
@@ -0,0 +1,36 @@
+//@compile-flags: -Zmiri-tree-borrows -Zmiri-permissive-provenance
+
+/// Checks how accesses from one subtree effect other subtrees.
```

Comment:
```suggestion
/// Checks how accesses from one subtree affect other subtrees.
```
This typo occurs quite a lot, I won't annotate it everywhere. :)